app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/AgendaItemDetailViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package com.andrew.tasky.presentation.agenda_item_detail
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class AgendaItemDetailViewModel : ViewModel() {
+
+    private val _isInEditMode = MutableStateFlow(false)
+    val isInEditMode = _isInEditMode.asStateFlow()
+    fun setEditMode(x: Boolean){
```

Comment:
I'd avoid using variable names like `x`. Even though, in this case it's obvious, it's always better to have a name that tells someone what it stands for like `isEditing` which also makes it easier to scan such code without first needing to think "Uh what was that for again"